zellij-utils/src/input/theme.rs

```diff
@@ -23,11 +30,29 @@ pub struct FrameConfigFromYaml {
 }
 
 #[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
-struct Theme {
+struct ThemeFromYaml {
+    palette: PaletteFromYaml,
+}
+
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
+pub struct Theme {
     #[serde(flatten)]
     palette: PaletteFromYaml,
 }
 
+#[derive(Debug, Error)]
+pub enum ThemeError {
```

Comment:
There are similar error types in `ConfigError`, is it beneficial to have a separate `ThemeError`?

Reply:
Yes, that's right. In fact, there is no immediate benefit. Just, separated for the future.